src/LanguageServer/Impl/LanguageServer.cs

```diff
@@ -44,7 +45,6 @@ public sealed partial class LanguageServer : IDisposable {
 
         private IServiceContainer _services;
         private IUIService _ui;
-        private ITelemetryService2 _telemetry;
```

Comment:
Is there a reason why you're removing this? The RPC trace listener isn't going to be the only thing that needs to access telemetry (and I don't think it has anything to do with this PR's intent).

Reply:
Well, I don't like fields that are just hanging around.

Reply:
I think Jake is now using it ;-)

Reply:
He moved the `GetService` down into the constructor call for the class I added, so it'll still function correctly. My statement was more about how if/when we start sending more telemetry than just for RPC method exceptions, then this change will just be undone anyway.

Reply:
Not exactly. If all the usages of `ITelemetryService2` instance in `LanguageServer` is limited to passing it to another constructors, we won't need a field as well. We will need a field only if `LanguageServer` actually uses the `ITelemetryService2`, but for now it isn't used.